revenuecat_examples/purchase_tester/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -7,10 +7,8 @@ import Foundation
 
 import app_links
 import purchases_flutter
-import purchases_ui_flutter
 
 func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
   AppLinksMacosPlugin.register(with: registry.registrar(forPlugin: "AppLinksMacosPlugin"))
   PurchasesFlutterPlugin.register(with: registry.registrar(forPlugin: "PurchasesFlutterPlugin"))
-  PurchasesUiFlutterPlugin.register(with: registry.registrar(forPlugin: "PurchasesUiFlutterPlugin"))
```

Comment:
This happens locally for me as well but I wasn't sure whether to check it in github since st seems like it should be added right? I think it might be due to the UI plugin not working on macos?